_data/rules.yml

```diff
@@ -360,3 +360,21 @@ services:
             description: SSL certificate has expired already
             query: 'probe_ssl_earliest_cert_expiry - time()  <= 0'
             severity: error
+
+  - name: Juniper
+    exporters: 
+      - name: czerwonk/junos_exporter
+        doc_url: https://github.com/czerwonk/junos_exporter
+        rules:
+        - name: Switch is down
+          description: The switch appears to be down
+          query: junos_up == 0
+          severity: critical
+        - name: High Bandwith Usage 1GiB
+          description: Interfaceis highly saturated for at least 2 min. (> 0.95GiB/s)
+          query: 'irate(junos_interface_transmit_bytes[2m]) * 8 > 9.5e+09'
+          severity: warning
+        - name: High Bandwith Usage 10GiB
```

Comment:
Thanks for your contribution @AngelFreak. I never heard about Juniper before!

I'm not sure it is necessary to duplicate for 10GB and 1GB interfaces.
Or I would prefer a duplication to differentiate criticity: a 95% threshold in critical + 80% threshold in warning.

I think you reversed `9.5e+08` and `9.5e+09` for 10GB and 1GB ifaces. 😄

Can we change `9.5e+08` by `1e+09 * 0.95` to facilitate reading ?

WDYT ?

Reply:
Thanks for pointing it out, i agree that the notation should be more readable.
I will make the changes tomorrow :)

Reply:
Will you please verity the new notation and the expected outcome ?
Just to make sure i did not make any calculation errors.

Reply:
I wonder if it should be 1e+10 for a 10Gbit interface?

Reply:
1e+09 is for 1GB 😄